ros/src/waypoint_updater/waypnt_updater.py

```diff
@@ -569,24 +576,29 @@ def set_stopped(self, start_ptr, num_wps):
             self.waypoints[mod_ptr].JMT_ptr = -1
         # cleanup jmt stack
         self.JMT_List[:] = []
+        # won't match self.next_tl_wp when reset
+        self.stop_target = 0
 
     def set_transition_to_stop(self, mod_ptr):
         # gracefully slow down to stopped at end of jmt decel curve
         disp = self.waypoints[mod_ptr].get_s() -\
             self.waypoints[mod_ptr-1].get_s()
 
-        decel = max(self.waypoints[mod_ptr-1].JMTD.A * 0.5, 0.5)
+        decel = min(self.waypoints[mod_ptr-1].JMTD.A * 0.5, -0.25)
```

Comment:
this was backwards since deceleration is negative. Setting the deceleration in this transition area only has an effect if the car is actually not yet quite stopped, when the TL signal changes to go.  In which case the speedup curve is moderated.